### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Bavon101**
- [ ] Add Desktop geometry images to src
- [ ] Add desktop styling using media query
- [ ] Add hover animation to display project card
- [ ] Add desktop view in html sections

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,13 +10,17 @@
...
 
 	<body class="body">
-		<header>
-			<div class="header">
-
-				<img src="src/img/svg/image_geometry_1.svg" alt="geometric image" class="image_header">
-				<nav><button class="ic_image">
-					</button></nav>
-			</div>
+		<header class="header">
```
Comment (Meltrust) [inline index.html:23]: - [x] It seems that your fonts have not been imported.  That is why although you are declaring them on the CSS elements, they are not displaying correctly.  

Kindly go [here](https://fonts.google.com/specimen/Poppins), and choose all the font weights that you will need.  Then, on the right, you will get 3 `<links>`'s.  Paste them to the `<head>` section of your HTML file!

Do this for all the fonts that you need, not just "Poppins".  Look:

image

Paste the 3 `<link>`'s here:

image


That way, all the fonts that you declared will work and actually render on the page 💪

File: index.html [hunk 2]
```diff
@@ -34,233 +38,314 @@ <h1 class="name">
...
 		</section>
 
 		<section class="project_section">
 			<h2 class="projects_label">
 				Projects
 			</h2>
-			<div class="project_card">
-				<div class="project_image"></div>
+			<div class="image_dot_1"></div>
+			<div class="project_card_macro">
```
Comment (Meltrust) [inline index.html:60]: - [ ] Impressive animation! Now let's align these popup language tags vertically centered like in the design. 💪 

image

File: index.html [hunk 3]
```diff
@@ -34,233 +38,314 @@ <h1 class="name">
...
+							</ul>
 					</div>
 					<button class="see_project">
 						See this project
 						<i class="arrow_forward"></i>
 					</button>
 				</div>
 			</div>
-			<div class="project_card">
+			<div class="project_section_images">
```
Comment (Meltrust) [inline index.html]: - [x] Some of these images are not aligned like in the Figma:

Your Implementation | Figma Design
:---:|:---:
image | image

Please kindly correct them, so they look like the design 💯

File: index.html [hunk 4]
```diff
@@ -280,63 +365,70 @@ <h3 class="about_title">
...
+						<li class="my_langaues_text_style">Terminal</li>
+					</ul>
+				</div>
+				<div class="dot_image_with_teal"></div>
 			</div>
 		</section>
 		<section class="contact_section">
 			<div class="geometry_image"></div>
-			<h3 class="about_title">
+			<h3 class="contact_title">
```
Comment (Meltrust) [inline index.html:529]: - [x] This section's background color is different than the one given in the Figma.  Let's correct it, so your project looks more consistent with its colors.

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @Bavon101 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Your project is really beautiful 👏 , you are really close to perfection.  Just implement the small changes below in the comments and you will get it approved in no time!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 5]
```diff
@@ -551,6 +1216,16 @@
...
+@media (min-width: 992px) {
+  .collaboration_form_button {
+    position: absolute;
+    width: 254px;
+    height: 70px;
+    left: 593px;
+    top: 650px;
+  }
+}
+
```
Comment (rustam-tolipov) [inline styles.css:1228]: ## WELL DONE ✅😲
- [ ] [OPTIONAL] All the things are fixed from the last review 👏😀. I also have a little suggestion to help improve your design. I recommend changing the cursor to pointer 👆 when hovering buttons. **TIP 📌**: You can add `cursor: pointer:`

#### Image example from figma
Buttons Primary Dark

**Review by rustam-tolipov — APPROVED** [review]
## Hi @Bavon101  👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**